comp/process/forwarders/forwardersimpl/forwarders.go

```diff
@@ -73,7 +73,7 @@ func newForwarders(deps dependencies) (forwarders.Component, error) {
 }
 
 func createForwarder(deps dependencies, options *defaultforwarder.Options) defaultforwarder.Component {
-	return defaultforwarder.NewForwarder(deps.Config, deps.Logger, deps.Lc, false, options, false).Comp
+	return defaultforwarder.NewForwarder(deps.Config, deps.Logger, deps.Lc, false, options).Comp
```

Comment:
Is this still correct after your change because `dep.Params.useNoopForwarder` defaults to false as it's a boolean?